docs/core/porting/libraries.md

```diff
@@ -13,336 +13,183 @@ ms.assetid: a0fd860d-d6b6-4659-b325-8a6e6f5fa4a1
+AppDomains isolate apps from one another. AppDomains require runtime support and are generally quite expensive. They're not implemented in .NET Core. We don't plan on adding this capability in future.
 
-### App Domains
+For code isolation, we recommend separate processes or using containers as an alternative. For the dynamic loading of assemblies, we recommend the new <xref:System.Runtime.Loader.AssemblyLoadContext> class. Information (such as the name and base directory) is provided by APIs on other types, for instance <xref:AppContext.BaseDirectory%2A>. Some scenarios, such as getting the list of loaded assemblies, are unsupported, as they're inherently fragile.
 
-AppDomains can be used for different purposes on the .NET Framework. For code isolation, we recommend separate processes and/or containers as an alternative. For dynamic loading of assemblies, we recommend the new @System.Runtime.Loader.AssemblyLoadContext class.
+To make code migration from .NET Framework easier, we've exposed some of the <xref:System.AppDomain> API surface in .NET Core. Some of the APIs function normally (for example, <xref:System.AppDomain.UnhandledException?displayProperty=fullName>), some of them do nothing (for example, <xref:System.AppDomain.SetCachePath%2A>), and some of them throw <xref:System.PlatformNotSupportedException> (for example, <xref:System.AppDomain.CreateDomain%2A>).
 
 ### Remoting
 
-For communication across processes, inter-process communication (IPC) mechanisms can be used as an alternative to Remoting, such as [Pipes](https://docs.microsoft.com/dotnet/core/api/system.io.pipes) or [Memory Mapped Files](https://docs.microsoft.com/dotnet/core/api/system.io.memorymappedfiles.memorymappedfile).
-
-Across machines, you can use a network based solution as an alternative, preferably a low-overhead plain text protocol such as HTTP. [KestrelHttpServer](https://github.com/aspnet/KestrelHttpServer), the web server used by ASP.NET Core, is an option here. Remote proxy generation via [Castle.Core](https://github.com/castleproject/Core) is also an option to consider.
-
-### Binary Serialization
-
-As an alternative to Binary Serialization, there are multiple different serialization technologies to choose. You should choose one that fits your goals for formatting and footprint. Popular choices include:
-
-* [JSON.NET](http://www.newtonsoft.com/json) for JSON
-* @System.Runtime.Serialization.DataContractSerializer for both XML and JSON
-* @System.Xml.Serialization.XmlSerializer for XML
-* [protobuf-net](https://github.com/mgravell/protobuf-net) for Protocol Buffers
+.NET Remoting was identified as a problematic architecture. It's used for cross-AppDomain communication, which is no longer supported. Also, Remoting requires runtime support, which is expensive to maintain. For these reasons, .NET Remoting isn't supported on .NET Core.
```

Comment:
For consistency, this should add the sentence "we don't plan on adding support in the future" as you did above, if it applies.